airgun/widgets.py

```diff
@@ -473,6 +516,58 @@ def fill(self, values):
         self.new_parameter_value.fill(values['value'])
 
 
+class TableActionList(Widget):
```

Comment:
What's the difference with `SelectActionList` and widgetastic_patternfly's `Dropdown`? Seems like we're starting to introduce very similar separate widgets for the same purpose

Reply:
1) You probably meant `TableActionList`
2) My widget operate with separate button + dropdown, their with dropdown
3) I substituted 150 lines of redundant code with
```
self.open_dropdown.click()
self.browser.click(
    self.browser.element(self.dropdown_action.format(value)))
``` 
In future we can use my approach as I don't want do 100 actions and waste time on unnecessary waiters, closing dropdowns and etc